client/components/forms/form-text-input/index.jsx

```diff
@@ -1,16 +1,16 @@
 /**
  * External dependencies
  */
-var React = require( 'react/addons' ),
-	classnames = require( 'classnames' ),
-	omit = require( 'lodash/object/omit' ),
-	classNames = require( 'classnames' );
+import React from 'react/addons';
```

Comment:
This module doesn't use addons. Can we remove this `/addons` suffix while we're already here?